Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -244,8 +244,7 @@ void JitArm64::fcmpX(UGeckoInstruction inst)
 
 	SetJumpTarget(pNaN);
 
-	ORR(XA, XA, 64 - 61, 0, true);
-	ORR(XA, XA, 0, 0, true);
+	MOVI2R(XA, PPCCRToInternal(0x1));
```

Comment:
The other calls could potentially be moved over as well. Would need to check the difference in code though, it would most likely end up being less efficient since MOVI2R doesn't cover all cases to be the most efficient.